Feature display names

zaverbac opened the pull request.

mrjoshua520 committed:
feat: :sparkles: Add Display Names page

Adds a svelte file for displaying names coming from the database

zaverbac committed:
feat(client): :sparkles: Add firestore add & get functions

Add firestore getUsers & addUser to firebase emulator. Add seperate ts file for firebase functions

njumaah reviewed (dismissed):
please make sure to not use caps for filenames and urls.

nstringham committed:
fix: 🐛 read firestore data correctly

zaverbac committed:
fix(firebaseFunctions): :bug: Export getUsers function instead of names fixed no display of names

Export getUsers function instead of names & move names to page.svelte to fix no names being displayed on webpage

zaverbac committed:
fix(firebaseFunctions): :bug: Fix names not displaying on webpage

Export getUsers function instead of names & move names to page.svelte to fix names not displaying
